test/functional/cmdLineTests/jitserver/build.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+
+<!--
+  Copyright IBM Corp. and others 2022
```

Comment:
2023?

Reply:
That's the copyright start date, we no longer need to add an end date.

Reply:
Ah I see, new file. :)